docs: CI Integrations: design additional parameter to skip-coverage (#1009)
platform-platform/monorepo: metrics/ci_integrations/docs/02_ci_integration_user_guide.md

ci_integrations sync --config-file="path/to/config_file.yaml"
```

#### Build coverage importing

> By default, build coverage data is fetching along with builds. 

Reviewer: ```suggestion
By default, the CI Integrations tool fetches coverage data for each build. This coverage data comes from an appropriate coverage artifact published during the CI build. The algorithm fetches artifacts and looks for the required coverage one and then downloads it for parsing. This is a mandatory step, and if fetching or downloading an artifact fails - the synchronization process fails as well. 
```

> If the given `Access Token` has no permission to read build artifacts in CI the `sync` command will fail. To skip fetching the coverage step, you should use the `sync` command with the `--skip-coverage` flag.

Reviewer: ```suggestion
If your CI project is not ready to publish coverage data, or if you for some reason want to skip this step and disable coverage fetching, you may use the `--skip-coverage` flag. This flag disables any artifacts fetching and manipulating for the sync process. Consider the following example of using the `--skip-coverage` flag:
```

> For example:
> 

Reviewer: ```suggestion
```

```bash
ci_integrations sync --config-file="path/to/config_file.yaml" --skip-coverage
```

> 

Reviewer: ```suggestion
_**Note**: All builds synchronized with the `--skip-coverage` flag won't contain any coverage information. This may affect the coverage metric available on the Metrics Web Application._

```

> #### Automate CI Integrations

Reviewer: ```suggestion
#### Automating CI Integrations
```
To make it match the `Making Things Work` section title.
